gradle/libs.versions.toml

```diff
@@ -107,6 +107,7 @@ wordpress-rs = 'alpha-20250411'
 wordpress-utils = '3.14.0'
 automattic-ucrop = '2.2.11'
 zendesk = '5.4.0'
+turbine = '1.2.0'
```

Comment:
@adalpari I've not heard of Turbine before, but it looks useful. Was this something you used for Tumblr?